llvm/lib/Transforms/IPO/MergeFunctions.cpp

```diff
@@ -467,9 +489,16 @@ bool MergeFunctions::runOnModule(Module &M) {
   return Changed;
 }
 
+std::pair<bool, DenseMap<Function *, Function *>>
+MergeFunctions::runOnFunctions(std::set<Function *> &F) {
+  bool MergeResult = false;
+  MergeResult = this->run(F);
+  return {MergeResult, this->DelToNewMap};
```

Comment:
Do you need both the boolean and the map here? Is the boolean equivalent to a non-empty map?

Reply:
Resolved in [350395d](https://github.com/llvm/llvm-project/pull/111045/commits/350395d7c6e7614e3012443abe155cfc1b307e87).